server/db/migrate/20150216190947_add_rhev_params.rb

```diff
@@ -0,0 +1,16 @@
+class AddRhevParams < ActiveRecord::Migration
```

Comment:
Ok I can focus on doing that. Now that I know how to fix the DB using Rails console :smile: